BUILD

```diff
@@ -16,3 +16,8 @@ envoy_cc_library(
     srcs = ["version_generated.cc"],
     deps = ["//source/common/common:version_includes"],
 )
+
+config_setting(
+    name = "force_test_link_static",
+    values = {"define": "FORCE_TEST_LINK_STATIC=yes"},
```

Comment:
It's case sensitive, but I don't think there's much convention here, in fact `--define` is somewhat discouraged (see https://bazel.build/versions/master/docs/be/general.html). We could use this with `--config` instead, but I was planning on using that for things like ASAN, so it's good to have an orthogonal flag.